### PR TITLE
Add RELEASE.md to document release process

### PR DESCRIPTION
**Pull request by touilleMan**

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
+# Release cheatsheet
+
+## Major/minor vs patch versions
+
+When the need for release come, the obvious question is "what is going to be
+the next version ?".
```
Comment (FirelightFlagboy) [inline RELEASE.md:6]:
```suggestion
When the need for release come, the obvious question is 

> what is going to be the next version ?.
```

Comment (touilleMan) [reply RELEASE.md]: :+1:

**Review by FirelightFlagboy — APPROVED** [review]

**Review by Max-7 — APPROVED** [review]

File: docs/conf.py [hunk 2]
```diff
@@ -53,16 +53,16 @@
 
 # General information about the project.
 project = "Parsec"
-copyright = "2016-2020, Scille SAS"
+copyright = "2016-present, Scille SAS"
```
Comment (FirelightFlagboy) [inline docs/conf.py:56]: why not `future` ? :P

**Review by FirelightFlagboy — APPROVED** [review]

File: docs/conf.py (hunk 2 above)
Comment (touilleMan) [reply docs/conf.py:56]: image

Comment (touilleMan) [reply docs/conf.py:56]: on a more serious note, this is what is done in facebook open source projects, for instance:
- https://github.com/facebookincubator/katran/blob/2121b31994a4200f442bc77480d5c9677f6cbbcc/katran/lib/BpfAdapter.cpp#L1
- https://github.com/facebookincubator/SocketRocket/blob/6f61b5437eaf425e073309f7252c8fa5ea908311/SocketRocket/SRSecurityPolicy.m#L2

So if it's corporate enough for *evil corp®*, it's must for us too :trollface:

**Pull request merged**